### PR TITLE
Fake items (fake unbox and trade messages)

### PR DESCRIPTION
**Pull request by DoomFishWasTaken**

### CODE REVIEW

**Comment by KernCS** [comment]
Reset Name dosen work? https://i.imgur.com/C6QfJpB.png
"Fake Item Flags" Crashing ?

**Comment by unnamed01X** [comment]
> Reset Name doesn't work? https://i.imgur.com/C6QfJpB.png
> "Fake Item Flags" Crashing ?

same ://

**Comment by DoomFishWasTaken** [comment]
@KernCS does the star option cause a crash?

**Comment by unnamed01X** [comment]
> @KernCS does the star option cause a crash?

whole thing "fake item flags" when i try to open that

**Comment by KernCS** [comment]
> @KernCS does the star option cause a crash?

as unnamed01X Said it is the whole "fake item flags" thing

**Comment by DoomFishWasTaken** [comment]
ye I'm on it rn

**Comment by unnamed01X** [comment]
@DoomFishWasTaken it would be also nice if you add the "circle" thing before the *text name*, because in the real unbox there are couple of circles, like that, example: 
Screenshot_2

**Comment by DoomFishWasTaken** [comment]
[enjoy :)](https://i.imgur.com/2tAT8bY.png)

**Comment by DoomFishWasTaken** [comment]
also @unnamed01X added those circles

**Comment by unnamed01X** [comment]
yeah, i see, i'll try it, thanks a lot bro, give me your discord contact, i would like to have more people like you there 🥇 <3

**Comment by KernCS** [comment]
Nice Works perfect now https://i.imgur.com/gPRN1vN.png

**Comment by unnamed01X** [comment]
@DoomFishWasTaken it's still buggy, i mean, if u "take" the P90 for example, you'll get pp-bizon on the chat, same thing with the new knives or gloves, showing as UMP or other weapons xd wtf

**Comment by GreenNightfall** [comment]
How can I add the feature to the cheat? When I try it manually it causes a lot of errors...
So can download the overwritten files somwhere?

**Comment by KernCS** [comment]
> How can I add the feature to the cheat? When I try it manually it causes a lot of errors...
> So can download the overwritten files somwhere?

what kind of errors are you getting?

**Comment by TheFantasticLoki** [comment]
can you update this so it doesn't conflict with the master.

**Comment by RichhieCheaterfun** [comment]
Hi guys, anyone still have this (as .dll) ?

I would be really grateful to him.

**Comment by techprimedev** [comment]
Cause Crash on Fake item.

**Comment by DoomFishWasTaken** [comment]
decided to update this :)

**Comment by feezytuber** [comment]
Yeah, how do I put all this together? I need a tutorial, because I do not know what to do. Either that, or you can send me a DLL. Thanks.

**Comment by feezytuber** [comment]
I dunno how to do anything with these files. I need a guide through

**Comment by ahmet1231** [comment]
how to download files

File: Osiris/Hacks/Misc.cpp [hunk 1]
```diff
@@ -428,10 +428,227 @@ void Misc::fakeBan(bool set) noexcept
...
+            item = "Moto Gloves";
+        else if (config->misc.fakeItemType == 52)
+            item = "Specialist Gloves";
+        else if (config->misc.fakeItemType == 53)
+            item = "Sport Gloves";
+        else if (config->misc.fakeItemType == 54)
+            item = "Bloodhound Gloves";
+        else if (config->misc.fakeItemType == 55)
+            item = "Hydra Gloves";
+        else if (config->misc.fakeItemType == 56)
```
Comment (mkot2) [inline Osiris/Hacks/Misc.cpp:618]: lmao

Comment (DoomFishWasTaken) [reply Osiris/Hacks/Misc.cpp:618]: ikr lol

**Comment by YeetingWaterbottle** [comment]
thinking of re-updating the code? I tried fixing it myself but I'm not big brain enough.

**Review by AsukiiJapan — COMMENTED** [review]
this is awesome keep it goin!

**Pull request closed without merging**